lib/server/handlers/sthGetDataHandler.js

```diff
@@ -62,8 +62,13 @@ function getRawData(request, reply) {
     );
     sthDatabase.getCollection(query, sthOptions, function(err, collection) {
         if (err) {
-            // The collection does not exist, reply with en empty response
-            sthLogger.warn(
+            if(err.name === 'MongoNetworkError' && err.message.includes('failed to connect to server')){
```

Comment:
Aligned with https://github.com/telefonicaid/fiware-sth-comet/pull/583/files#r813167242, maybe just:

```suggestion
            if(err.name === 'Not connected')){
```